src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,31 @@
+import classes from './Home.module.scss';
+import Sidebar from 'src/components/Sidebar/Sidebar';
+import Feed from 'src/components/Feed/Feed';
+
+const Home = () => {
+  return (
+    <div className={`${classes['c-home']}`}>
+      <Sidebar>
+        <div>
+          <h1>User Info</h1>
+        </div>
+        <div>
+          <h2>Menu</h2>
+        </div>
+      </Sidebar>
```

Comment:
Since you decided for this approach you should create `<SidebarItem />` component and replace these divs since at some point you will create stylings for this divs and then you'll need to repeat them couple of times, and on all of that that stile would be in Home scss file and that is not what we want 😄